sdk/src/claim.rs

```diff
@@ -2497,12 +2497,17 @@ impl Claim {
                                     if let Some(readaction_label) =
                                         assertion_label_from_uri(&redacted_uri)
                                     {
-                                        let (label, instance) =
-                                            Claim::assertion_label_from_link(&readaction_label);
+                                        // The assertion may or may not be in the assertion store.
+                                        // It can exist and be zeroed or be removed entirely
+                                        // but it must be in the claim's assertions HashUri list
                                         parent_tested = Some(
-                                            ingredient.get_assertion(&label, instance).is_some(),
+                                            ingredient
+                                                .assertions()
+                                                .iter()
+                                                .any(|a| a.url().contains(&readaction_label)),
```

Comment:
Is the redaction label you are passing in the full URL?  That is what is expected.

Reply:
You are correct.  A redaction will be in in the assertion list but not the assertion store.  Good catch.

Reply:
maybe make this a Claim fuction, perhaps **fn has_assertion_entry() -> bool**